google/cloud/firestore_v1/async_query.py

```diff
@@ -126,6 +127,47 @@ def __init__(
             recursive=recursive,
         )
 
+    async def chunkify(
+        self, chunk_size: int
+    ) -> AsyncGenerator[List[DocumentSnapshot], None]:
+        # Catch the edge case where a developer writes the following:
+        # `my_query.limit(500).chunkify(1000)`, which ultimately nullifies any
```

Comment:
This comment seems to be the opposite of the check `chunk_size < self._limit` (chunk size is larger in sample than limit)

Reply:
Oh my goodness. I reversed this today in a moment of what I now see was confusion. Never was good at left and right, either.

Reply:
I even had the tests flipped 😭